db/schema.rb

```diff
@@ -25,8 +25,8 @@
   end
 
   create_table "rents", force: :cascade do |t|
-    t.bigint "user_id", null: false
-    t.bigint "book_id", null: false
+    t.bigint "user_id"
```

Comment:
Why we had this `null:false` values? :thinking: we don't have a migration to this